packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -1069,17 +1075,27 @@ public function import_comment( $comment, $post_id, $post_just_imported = false
 		/**
 		 * Post processing completed.
 		 *
-		 * @param int $post_id New post ID.
+		 * @param int $comment_id New comment ID.
 		 * @param array $comment Raw data imported for the comment.
-		 * @param array $meta Raw meta data, already processed by {@see process_post_meta}.
 		 * @param array $post_id Parent post ID.
 		 */
 		do_action( 'wxr_importer_processed_comment', $comment_id, $comment, $post_id );
 	}
 
 	public function import_comment_meta( $meta_item, $comment_id ) {
-		$value = maybe_unserialize( $meta_item['value'] );
-		add_comment_meta( $comment_id, wp_slash( $meta_item['key'] ), wp_slash( $value ) );
+		$meta_item = apply_filters( 'wxr_importer_pre_process_comment_meta', $meta_item, $comment_id );
+		if ( empty( $meta_item ) ) {
+			return false;
+		}
+
+		if ( ! isset( $meta_item['comment_id'] ) ) {
+			$meta_item['comment_id'] = $comment_id;
+		}
+
+		$value           = maybe_unserialize( $meta_item['meta_value'] );
```

Comment:
Are we unserializing it only for `add_comment_meta` to serialize it again? Is there any way we could avoid doing that work? I suppose not, since `add_comment_meta` likely triggers a bunch of hooks – although I wonder if we should disable all the content insertion hooks during the import 🤔 

Also, should we worry about `__wakeup` code execution vulnerabilities and such? Or would WordPress call this function on that value after inserting it into the database anyway?